app/gui/view/component-browser/component-list-panel/src/navigator/entry.rs

```diff
@@ -0,0 +1,204 @@
+//! A module containing the grid view entry type which represents an arbitrary icon.
```

Comment:
The section navigator now uses the grid view instead of the list view. It means the whole component browser is not utilizing any list views anymore!